packages/app/src/components/add-tag/AddTagStepInfo.vue

```diff
@@ -48,52 +44,48 @@ export default defineComponent({
   emits: {
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
     'update:nfc-tag': (_payload: NFCTag): boolean => true,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    'update:is-valid': (_payload: boolean): boolean => true,
   },
   setup(props, ctx) {
     const currentTag = ref(props.nfcTag);
```

Comment:
@schnickalodeon Do you know why we had this in here? Just noticed it and it seems completely unnecessary to me.